modules/git/tree_entry.go

```diff
@@ -11,6 +11,10 @@ import (
 	"strings"
 )
 
+func (te *TreeEntry) Repo() *Repository {
+	return te.ptree.repo
+}
```

Comment:
Annoyingly necessary to punch through the package-private variables here due to the lines in the previous file..